src/index.html

```diff
@@ -9,7 +9,7 @@
   <link rel="stylesheet" href="assets/css/bootstrap-base.css">
   <!-- jQuery library -->
   <script src="//ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
-  <script src="assets/chart.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.4.0/Chart.bundle.min.js"></script>
```

Comment:
@nikhilrayaprolu I'm not able to find `chart.bundle.min.js` file. Only CDN link was available.